apiserver/params/model.go

```diff
@@ -139,7 +139,7 @@ type ModelInfo struct {
 	Life Life `json:"life"`
 
 	// Status is the current status of the model.
-	Status EntityStatus `json:"status"`
+	Status *EntityStatus `json:"status,omitempty"`
```

Comment:
Indeed. I'll change the other params I've made optional too. Thank you for pick up \o/